tests/src/test/scala/tests/commands/ExportCommandSuite.scala

```diff
@@ -51,7 +51,7 @@ class ExportCommandSuite extends tests.BaseSuite with tests.ConfigSyntax {
          |  'org.slf4j:slf4j-log4j12:1.6.4' is a transitive dependency of slf4j-1.6.4.
          |  - dependency: org.slf4j:slf4j-log4j12:1.6.1
          |                ^
-         |warning: 1 declared dependency was evicted.
+         |warning: 1 warning(s) found.
```

Comment:
This warning message seems less helpful now, will the user be getting a more informative message elsewhere?

Reply:
Previously all warnings were eviction warnings, but now there might be either transitive URL warning or eviction warning. I think it's clear from the warning message itself.